catalogue/webapp/pages/_error.tsx

```diff
@@ -25,16 +25,13 @@ const Page: NextPage<Props> = ({
   );
 };
 
-export const getServerSideProps: GetServerSideProps<Props> = async context => {
+Page.getInitialProps = async context => {
```

Comment:
Is it still necessary to do this?

Reply:
Rather than `export const getInitialProps`? I _think_ so in order to get the correct typing, yes (because it doesn't look like `GetInitialProps` isn't an export from `next`)

Reply:
But given we haven't updated next, which causes this issue, why have we had to update it?

It feels like a bit of a hack, and I'd rather approach it when updating next and we can try to understand it more fully as I'm not sure I'd understand this (and know that the typing in `getGlobalContextData` is coupled to this) coming back to it in the future.

Reply:
I believe this was a real issue prior to the updating of Next, and is the reason why our error pages currently aren't working as expected. [We can't use `getServerSideProps`](https://github.com/vercel/next.js/discussions/11945#discussioncomment-6790) because `_error` is statically exported in all versions of Next 10. We could use `getStaticProps` for the error page, but we're relying on things coming from the server (e.g. status code) so we either rethink what we want to display on the error page or we use `getInitialProps` I think.

Reply:
Rethinking it I think makes sense (note the general trend to have a [very very simple error pages](http://www.bbc.co.uk/oasiuj) as it seems mega impolite to ask people do download loads for nothing).

Not part of this PR though, could we leave a note, especially around the coupling to the `GlobalContextProvider`?